### ISSUE TITLE
No pretrained mode

### ISSUE THREAD

**Issue by JudasDie** [LiuShuaiyr/UAVMOT#16]
Hi, thanks for the work. Why there is no checkpoint provided for evaluation?

**Comment by LiuShuaiyr** [comment]
There are some issues with the current code version, and the test results file is provided for verification. Recently, I am in the period of graduation and my energy is limited. In the later stage, we will provide model files together after completing the code.

**Comment by xiaocc612** [comment]
> There are some issues with the current code version, and the test results file is provided for verification. Recently, I am in the period of graduation and my energy is limited. In the later stage, we will provide model files together after completing the code.

Can I take a moment of your time to update the GBF loss function section?
